pkg/policy/repository.go

```diff
@@ -663,17 +663,26 @@ func (p *Repository) Empty() bool {
 	return p.NumRules() == 0
 }
 
+// TranslationResult contains the results of the rule translation
+type TranslationResult struct {
+	// NumToServicesRules is the number of ToServices rules processed while
+	// translating the rules
+	NumToServicesRules int
+}
+
 // TranslateRules traverses rules and applies provided translator to rules
-func (p *Repository) TranslateRules(translator Translator) error {
+func (p *Repository) TranslateRules(translator Translator) (error, *TranslationResult) {
```

Comment:
Can we return error as a second value? It's not written anywhere afaik, but it seems like this is the convention to follow.

Reply:
Scratch that, hound also complained about it, please change :)